app/javascript/packs/invoice_payment.js

```diff
@@ -0,0 +1,87 @@
+import { loadStripe } from '@stripe/stripe-js'
+import 'PaymentGateways/stripe/styles/stripe.scss'
+
+const style = {
+  base: {
+    color: '#32325d',
+    fontFamily: 'Arial, sans-serif',
+    fontSmoothing: 'antialiased',
+    fontSize: '16px',
+    '::placeholder': {
+      color: '#32325d'
+    }
+  },
+  invalid: {
+    fontFamily: 'Arial, sans-serif',
+    color: '#fa755a',
+    iconColor: '#fa755a'
+  }
+}
+
+document.addEventListener('DOMContentLoaded', async () => {
+  const dataset = document.querySelector('.stripe-form').dataset
+  const stripePublishableKey = dataset.publishableKey
+  const clientSecret = dataset.clientSecret
+  const form = document.querySelector('#payment-form')
+  const callbackForm = document.querySelector('#payment-callback-form')
+  const payButton = document.querySelector('#submit-payment')
+  const cardError = document.querySelector('#card-error')
+  const errorMsg = document.querySelector('#card-error')
+  const spinner = document.querySelector('#spinner')
+  const buttonText = document.querySelector('#button-text')
+
+  const stripe = await loadStripe(stripePublishableKey)
+  const elements = stripe.elements()
+  const card = elements.create('card', { style })
```

Comment:
Can we pass to stripe elements classes instead I think yes
https://stripe.com/docs/js/elements_object/create_element?type=card#elements_create-options

Reply:
@hallelujah 
In fact, we can totally omit to pass the style or any classes in the options object. It will use default stripe styles (injected from stripe script). 
If we want to allow customization, we need to discuss how to do it. We are not using a liquid template, but an `.erb`, so it's not going to be so easy to add an external stylesheet. I think customization of css should be a follow-up task.
WDYT?

Reply:
BTW, the styles we have in `app/javascript/src/PaymentGateways/stripe/styles/stripe.scss` are for the placement of the form in our page. The ones passed to `elements.create` affect only the _card element_ inside the form. So, no confict.